okio/src/main/java/okio/Buffer.java

```diff
@@ -760,11 +760,17 @@ String readUtf8Line(long newline) throws EOFException {
   }
 
   @Override public byte[] readByteArray(long byteCount) throws EOFException {
-    checkOffsetAndCount(size, 0, byteCount);
-    if (byteCount > Integer.MAX_VALUE) {
+    if (byteCount < 0) {
+      throw new IllegalArgumentException("byteCount < 0: " + byteCount);
+    } else if (byteCount > Integer.MAX_VALUE) {
```

Comment:
super minor nit: we typically don't put `else` after `throw` or `return` and instead just do separate `if` statements.

Reply:
Any particular reason for minor nit? Or just a coding convention ?